src/block/icon/style.js

```diff
@@ -154,7 +155,21 @@ export const createStyles = props => {
 	}
 
 	// Advanced Column Gap
-	styles.push( ...createResponsiveStyles( '.ugb-icon__content-wrapper', '%scolumnGap', 'columnGap', '%spx', props.attributes, { important: true } ) )
+	styles.push( {
+		'.ugb-icon__content-wrapper': {
+			columnGap: getValue( 'columnGap' ) !== '' ? appendImportant( getValue( 'columnGap' ) + 'px', true ) : undefined,
```

Comment:
`getValue` should be able to format this, and `appendImportant` should be able to handle blank values.

```suggestion
			columnGap: appendImportant( getValue( 'columnGap', '%spx' ) ),
```